compiler/src/iree/compiler/Codegen/Common/test/gpu_materialize_encoding_gfx942.mlir

```diff
@@ -1224,3 +1224,34 @@ func.func @set_encoding_with_layout() attributes {
 //  CHECK-SAME:     inner_tiles = [128, 16]
 //  CHECK-SAME:     tensor<255x513xf32> -> tensor<2x33x128x16xf32>
 //       CHECK:   flow.dispatch.tensor.store %[[PACK]], %[[RESULT_BINDING]]
+
+// -----
+
+//------------------------------------------------------------------------------
+// Negative tests. The pass should do nothing for these cases.
+//------------------------------------------------------------------------------
+
+// This test ensures that no side-effects happen/errors are thrown in case of
+// missing encoding information like indexing maps.
+
+#encoding = #iree_encoding.encoding<operand_index = 0, op_type = matmul, element_types = [f32, f32, f32],
```

Comment:
We can drop the `round_dims_to` field, because it does not impact the contraction dimension inference.

Reply:
Thanks, removed it.